src/question_paneldynamic.ts

```diff
@@ -428,6 +438,17 @@ export class QuestionPanelDynamicModel extends Question
   get locTemplateDescription(): LocalizableString {
     return this.template.locDescription;
   }
+  /**
+   * A Boolean expression that is evaluated against each panel. If the expression evaluates to `false`, the panel becomes hidden.
+   *
+   * A survey parses and runs all expressions on startup. If any values used in the expression change, the survey re-evaluates it.
```

Comment:
A survey parses and runs all expressions on startup. If any value used in the expression changes, the survey re-evaluates it.